mesonbuild/compilers/c.py

```diff
@@ -155,6 +155,7 @@ def get_options(self) -> 'MutableKeyedOptionDictType':
         if self.info.is_windows() or self.info.is_cygwin():
             opts.update({
                 OptionKey('winlibs', machine=self.for_machine, lang=self.language): coredata.UserArrayOption(
+                    'c_winlibs',
                     'Standard Win libraries to link against',
                     gnu_winlibs,
```

Comment:
This pattern is getting repeated a lot, maybe it makes more sense to add a protected helper to the base Compiler class so we can be a bit DRYer?